autoload/asyncomplete.vim

```diff
@@ -51,6 +51,18 @@ function! asyncomplete#register_source(info) abort
         return
     endif
 
+    try
```

Comment:
Out of curiosity why try catch instead of checking if function exits? Try catch may be hiding bugs.